<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Nicolas</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Universal parser to XML</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="statusBar">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="format">
         <string>%p%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/images/icons/settings-4.png</normaloff>:/icons/images/icons/settings-4.png</iconset>
     </property>
     <addaction name="actionSelect_parser_config"/>
     <addaction name="actionDebug_WD"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_folder"/>
    <addaction name="separator"/>
    <addaction name="actionExport_to"/>
    <addaction name="separator"/>
    <addaction name="menuSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGenerate_map_sample"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpen_folder"/>
   <addaction name="actionExport_to"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_parser_config"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/icons/folder-14.png</normaloff>:/icons/images/icons/folder-14.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_to">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/icons/save.png</normaloff>:/icons/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Export to XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/icons/exit-2.png</normaloff>:/icons/images/icons/exit-2.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/icons/letter-i.png</normaloff>:/icons/images/icons/letter-i.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionOpen_folder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/icons/archive.png</normaloff>:/icons/images/icons/archive.png</iconset>
   </property>
   <property name="text">
    <string>Perform folder</string>
   </property>
  </action>
  <action name="actionSelect_parser">
   <property name="text">
    <string>Select parser</string>
   </property>
  </action>
  <action name="actionSelect_parser_config">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/icons/settings-4.png</normaloff>:/icons/images/icons/settings-4.png</iconset>
   </property>
   <property name="text">
    <string>Select parser map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionGenerate_map_sample">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/icons/cloud-computing.png</normaloff>:/icons/images/icons/cloud-computing.png</iconset>
   </property>
   <property name="text">
    <string>Generate map sample</string>
   </property>
  </action>
  <action name="actionDebug_WD">
   <property name="text">
    <string>Debug WD</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
